packages/stores/src/mockStore.ts

```diff
@@ -212,6 +215,7 @@ const mock = (): TRootStore => {
             changeSelectedLanguage: jest.fn(),
             current_language: 'EN',
             is_network_online: false,
+            server_time: moment(),
```

Comment:
`moment` is a dev dependency in this package just for the type, We can't use it here in the `mockStore` function. Also we should use a default value here so our tests can be stable.

Reply:
@farzin-deriv Any suggestion on how change it?

Reply:
@mahdiyeh-deriv this should work : 
`jest.fn(() => new Date("2023-04-12T19:51:49.130Z"));`
or 
`jest.fn(() => new Date("Wed Apr 12 2023 23:22:31 GMT+0350"));`